dotcom-rendering/src/components/Treats.tsx

```diff
@@ -18,10 +18,12 @@ const TextTreat = ({
 	text,
 	linkTo,
 	borderColour,
+	textColour,
 }: {
 	text: string;
 	linkTo: string;
 	borderColour?: string;
+	textColour?: string;
```

Comment:
This feels like it shouldn't be optional, with the way its used on L43 it seems like you could end up with some broken CSS if this prop isn't set!

```css
color: ;
```